web/core/src/jsMain/kotlin/org/jetbrains/compose/web/attributes/EventsListenerScope.kt

```diff
@@ -5,191 +5,197 @@ import androidx.compose.web.events.SyntheticEvent
 import androidx.compose.web.events.SyntheticMouseEvent
 import androidx.compose.web.events.SyntheticWheelEvent
 import org.jetbrains.compose.web.events.*
-import org.w3c.dom.HTMLFormElement
 import org.w3c.dom.events.EventTarget
 
 private typealias SyntheticMouseEventListener = (SyntheticMouseEvent) -> Unit
 private typealias SyntheticWheelEventListener = (SyntheticWheelEvent) -> Unit
 private typealias SyntheticDragEventListener = (SyntheticDragEvent) -> Unit
 
+@Deprecated(
+    message = "Renamed to EventsListenerScopeBuilder",
+    replaceWith = ReplaceWith("EventsListenerScopeBuilder", "org.jetbrains.compose.web.attributes.EventsListenerScopeBuilder")
+)
+typealias EventsListenerBuilder = EventsListenerScopeBuilder
+
 /**
- * [EventsListenerBuilder] is used most often not directly but via [AttrsBuilder].
+ * [EventsListenerScope] is used most often not directly but via [AttrsScope].
  * Its purpose is to add events to the element. For all most frequently used events there
  * exist dedicated method. In case you need to support event that doesn't have such method,
  * use [addEventListener]
  */
-open class EventsListenerBuilder {
-
-    protected val listeners = mutableListOf<SyntheticEventListener<*>>()
-
+interface EventsListenerScope {
+    val listeners: List<SyntheticEventListener<*>>
```

Comment:
A question similar to @eymar's question about attributesMap - do we need to leave ` val listeners` as part of public contract?

Reply:
https://github.com/JetBrains/compose-jb/pull/1694#discussion_r790290835